test/api_integration/apis/saved_objects/create.js

```diff
@@ -47,30 +47,35 @@ export default function ({ getService }) {
         })
       ));
 
-      it('should return 503 and not create kibana index', async () => {
+      it('should return 200 and create kibana index', async () => {
         await supertest
           .post(`/api/saved_objects/visualization`)
           .send({
             attributes: {
               title: 'My favorite vis'
             }
           })
-          .expect(503)
+          .expect(200)
           .then(resp => {
             // loose uuid validation
+            expect(resp.body).to.have.property('id').match(/^[0-9a-f-]{36}$/);
+
+            // loose ISO8601 UTC time with milliseconds validation
+            expect(resp.body).to.have.property('updated_at').match(/^[\d-]{10}T[\d:\.]{12}Z$/);
+
             expect(resp.body).to.eql({
```

Comment:
__optional nit:__ not really better, but just another way of using `eql` with object that has properties that aren't know upfront:
```js
expect(resp.body).to.eql({
  ...resp.body,
  type: 'visualization',
  version: 1,
  attributes: {
    title: 'My favorite vis'
  }
});
```

Reply:
yeah, I really wanted to list all expected properties